tslint.json

```diff
@@ -10,7 +10,8 @@
               "no-var-requires": false,
               "prefer-const": false,
               "quotemark": [true, "single"],
-              "prefer-for-of": false
+              "prefer-for-of": false,
+              "triple-equals": false
```

Comment:
why ?

Reply:
https://www.impressivewebs.com/why-use-triple-equals-javascipt/

Reply:
This is part of a patch from a commit made in master to address the following issue:
https://github.com/daostack/subgraph/pull/473